### ISSUE TITLE
func-call-spacing reports "unexpected newline", when there is a space (not a newline)

### ISSUE THREAD

**Issue by ChumpChief** [eslint/eslint#13190]
Just suggesting an update to the error string.  Based on the comments in the rule, looks like there were intended to be three distinct messages, rather than just the two.

**Tell us about your environment**

* **ESLint Version:** 6.8.0
* **Node Version:** 12.13.0
* **npm Version:** 6.12.0

**What parser (default, Babel-ESLint, etc.) are you using?**
@typescript-eslint/parser

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
"func-call-spacing": "error"
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
console.log ("hi");
```

```bash
npm run eslint
```

**What did you expect to happen?**
error  Unexpected whitespace between function name and paren  func-call-spacing

**What actually happened? Please include the actual, raw output from ESLint.**
error  Unexpected newline between function name and paren  func-call-spacing

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by mdjermanovic** [comment]
Hi @ChumpChief, thanks for the bug report!

Confirmed in [online demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgZnVuYy1jYWxsLXNwYWNpbmc6IFwiZXJyb3JcIiovXG5cbmNvbnNvbGUubG9nIChcImhpXCIpO1xuIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo2LCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19). We should fix this, I'm working on it.
